tests/components/tomorrowio/test_weather.py

```diff
@@ -153,9 +153,66 @@ async def test_legacy_config_entry(hass: HomeAssistant) -> None:
     assert len(er.async_entries_for_config_entry(registry, entry.entry_id)) == 30
 
 
-async def test_v4_weather(hass: HomeAssistant) -> None:
+async def test_v4_weather(hass: HomeAssistant, tomorrowio_config_entry_update) -> None:
     """Test v4 weather data."""
     weather_state = await _setup(hass, API_V4_ENTRY_DATA)
+
+    tomorrowio_config_entry_update.assert_called_with(
+        [
+            "temperature",
+            "humidity",
+            "pressureSeaLevel",
+            "windSpeed",
+            "windDirection",
+            "weatherCode",
+            "visibility",
+            "pollutantO3",
+            "windGust",
+            "cloudCover",
+            "precipitationType",
+            "pollutantCO",
+            "mepIndex",
+            "mepHealthConcern",
+            "mepPrimaryPollutant",
+            "cloudBase",
+            "cloudCeiling",
+            "cloudCover",
+            "dewPoint",
+            "epaIndex",
+            "epaHealthConcern",
+            "epaPrimaryPollutant",
+            "temperatureApparent",
+            "fireIndex",
+            "pollutantNO2",
+            "pollutantO3",
+            "particulateMatter10",
+            "particulateMatter25",
+            "grassIndex",
+            "treeIndex",
+            "weedIndex",
+            "precipitationType",
+            "pressureSurfaceLevel",
+            "solarGHI",
+            "pollutantSO2",
+            "uvIndex",
+            "uvHealthConcern",
+            "windGust",
+        ],
+        [
+            "temperatureMin",
+            "temperatureMax",
+            "dewPoint",
+            "humidity",
```

Comment:
FWIW, `dewPoint` and `humidity` aren't very useful for forecasts other than hourly intervals (or less) since HA doesn't have min/max values for them and, even if it did, they still wouldn't be very useful since they won't necessarily coincide with the daily high/low temperatures. But to avoid extra API calls, this is necessary.

Reply:
gotcha, just another reason why this API can be so frustrating to use!